platform/android/java_godot_wrapper.cpp

```diff
@@ -331,11 +331,16 @@ void GodotJavaWrapper::init_input_devices() {
 	}
 }
 
-void GodotJavaWrapper::vibrate(int p_duration_ms) {
+void GodotJavaWrapper::vibrate(int p_duration_ms, float p_amplitude) {
 	if (_vibrate) {
 		JNIEnv *env = get_jni_env();
 		ERR_FAIL_NULL(env);
-		env->CallVoidMethod(godot_instance, _vibrate, p_duration_ms);
+
+		if (p_amplitude != -1.0) {
+			p_amplitude = CLAMP(p_amplitude, 0.0, 1.0);
+		}
+
+		env->CallVoidMethod(godot_instance, _vibrate, p_duration_ms, int(p_amplitude * 255));
```

Comment:
This should be:
```
int j_amplitude = -1.0;
if (p_amplitude != -1.0) {
  j_amplitude = CLAMP(int(p_amplitude * 255), 1, 255);
}

env->CallVoidMethod(godot_instance, _vibrate, p_duration_ms, j_amplitude);
```

This accomplishes a couple of things:
- If we get the default value (`-1`) we pass it as is to the java layer
- We're clamping the value between `1` and `255`. With the current logic in the PR, `0` could be passed to the java layer if `p_amplitude` is `0`, which [according to the documentation is not a supported value](https://developer.android.com/reference/android/os/VibrationEffect#createOneShot(long,%20int)).